site/data/dynamo.ts

```diff
@@ -164,6 +186,7 @@ export const insertPublishedDisruptionIntoDynamoAndUpdateDraft = async (
     ptSituationElement: PtSituationElement,
     disruption: Disruption,
     id: string,
+    status: string,
```

Comment:
This function will also need updating so that we don't write to the siri table in the case that the disruption is pending approval

Reply:
done